client/web/compose/src/components/PageBlocks/ChartBase.vue

```diff
@@ -4,11 +4,11 @@
     v-on="$listeners"
     @refreshBlock="refresh"
   >
-    <template>
+    <template v-if="options.liveFilterEnabled">
       <b-button
         rounded="full"
         variant="outline-light"
-        class="chart__livefilter-btn text-secondary d-print-none"
+        :class="['chart__livefilter-btn d-print-none position-absolute p-1', !!liveFilterValue ? 'text-primary' : 'text-secondary']"
         @click="liveFilterModal.show = true"
```

Comment:
When you open you should make a temporary copy of the value, so it doesn't affect the actual filter value even if you cancel. When you save you take that value and save it as the actual filter.